docs/BUILDING.md

```diff
@@ -55,6 +55,8 @@ Run `build.py --help` to see all options.
 1. Install OpenCV and FFmpeg, then modify `mediapipe/third_party/opencv_linux.BUILD` to make OpenCV visible to Bazel.
 2. Run `build.py desktop` to build desktop library.
 
+NOTE:- Bazel version 6.1.1 is strictly required and is installed automatically by running the `setup.py`, it is installed inside the Python venv.
```

Comment:
Nitpicking: No need to state required Bazel version explicitly if setup script has already handled for us, or we would need to update this section every time mediapipe change required version.

Reply:
So.... I think I should just remove this whole line altogether